pkg/types/openstack/validation/platform.go

```diff
@@ -31,24 +32,23 @@ func ValidatePlatform(p *openstack.Platform, n *types.Networking, fldPath *field
 		} else if !isValidValue(p.FlavorName, validFlavors) {
 			allErrs = append(allErrs, field.NotSupported(fldPath.Child("computeFlavor"), p.FlavorName, validFlavors))
 		}
+		p.TrunkSupport = "0"
 		netExts, err := fetcher.GetNetworkExtensionsAliases(p.Cloud)
 		if err != nil {
-			allErrs = append(allErrs, field.InternalError(fldPath.Child("trunkSupport"), errors.New("could not retrieve networking extension aliases")))
+			logrus.Warning("Could not retrieve networking extension aliases.")
```

Comment:
Can we indicate the result of that failure? E.g.
```
logrus.Warning("Could not retrieve networking extension aliases. Assuming trunk ports are not supported.")
```

Reply:
Ack.